_posts/0000-01-02-DreCampbell.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"Our deepest fear is not that we are inadequate. Our deepest fear is that we are powerful beyond measure."